src/convertToObject.js

```diff
@@ -12,7 +12,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convert = sourceString.split(';')
+    .map(removeEdgeGaps => removeEdgeGaps.trim())
```

Comment:
this is not a function, this is an element. Rename it, smth like this
```suggestion
    .map(string => string.trim())
```